kfs-web/src/main/webapp/static/xsd/purap/b2bPunchOutOrder.xsd

```diff
@@ -21,29 +21,122 @@
 <xsd:schema elementFormDefault="qualified"
 		    targetNamespace="http://www.kuali.org/kfs/purap/b2bPunchOutOrder"
 		    xmlns="http://www.kuali.org/kfs/purap/b2bPunchOutOrder" 
-		    xmlns:purap="http://www.kuali.org/kfs/purap/types"
-		    xmlns:kfs="http://www.kuali.org/kfs/sys/types"
 		    xmlns:xsd="http://www.w3.org/2001/XMLSchema">
 		    
-<xsd:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="../../xsd/sys/xml.xsd"/>
-<xsd:import namespace="http://www.kuali.org/kfs/purap/types" schemaLocation="../../xsd/purap/purapTypes.xsd" />
-<xsd:import namespace="http://www.kuali.org/kfs/sys/types" schemaLocation="../../xsd/sys/types.xsd" />
+<xsd:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/xml.xsd" />
+
+<!-- Simple Data Types from purapTypes.xsd -->
+
+    <xsd:simpleType name="domainType">
+        <xsd:restriction base="xsd:string">
+            <xsd:pattern value="DUNS|NetworkId|AribaNetworkUserId"/>
+        </xsd:restriction>
+    </xsd:simpleType>
+
+    <xsd:simpleType name="dunsNumberType">
+        <xsd:restriction base="xsd:string">
+            <xsd:maxLength value="50"/>
+        </xsd:restriction>
+    </xsd:simpleType>
+
+    <xsd:simpleType name="amountType">
```

Comment:
In order to get the xsd files to work properly, they could not simply be referenced from another xsd. This was copied directly from purapTypes.xsd, as-is, so I would assume that 4 decimals is correct.